application/backend/src/main/resources/apps/typerefinery/components/dialog/tabs/event/.content.json

```diff
@@ -0,0 +1,20 @@
+{
+    "sling:resourceType": "wcm/dialogs/components/tab",
+    "label": "Events",
+    "event": {
+        "sling:resourceType": "wcm/dialogs/components/multifield",
+        "name": "events",
+        "description": "Event that this component is emits/send to",
+        "label": "Events",
+        "key": {
+            "sling:resourceType": "wcm/dialogs/components/textfield",
+            "name": "key",
+            "label": "Topic"
+        },
+        "value": {
+            "sling:resourceType": "wcm/dialogs/components/textfield",
```

Comment:
This would be better to be local to a component with a dropdown of enabled functions. Will simplify language for this eg "Event | Can be any of the following: HIGHLIGHT or FILTER"

Reply:
after second component this will be hard to author as you wont know which "Event" apply to which component.

Reply:
This is used in the local component only.